product_docs/docs/pgd/5/terminology.mdx

```diff
@@ -2,83 +2,120 @@
 title: Terminology
 ---
 
-The terminology that follows is important for understanding EDB Postgres Distributed functionality and the requirements that it addresses in the realms of high availability, replication, and clustering.
+There are many terms you will come across in EDB Postgres Distributed that you may be unfamiliar with. This page is a list of many of those terms with quick definitions. 
 
-#### Asynchronous replication 
 
-Copies data to cluster members after the transaction completes on the origin node. Asynchronous replication can provide higher performance and lower latency than synchronous replication. However, it introduces the potential for conflicts because of multiple concurrent changes. You must manage any conflicts that arise.
+#### Asynchronous replication
 
-#### Availability
+Copies data to cluster members after the transaction completes on the origin node. Asynchronous replication can provide higher performance and lower latency than synchronous replication. However, asynchronous replication can see a lag in how long changes take to appear in the various cluster members. While the cluster will be [eventually consistent](#eventual-consistency), there will be potential for nodes to be apparently out of sync with each other.
 
-The probability that a system will operate satisfactorily at a given time when used in a stated environment. For many people, this is the overall amount of uptime versus downtime for an application. (See also [Nines](#nines))
+#### Commit scopes
+
+Rules for managing how transactions are committed between the nodes and groups of a PGD cluster. Used to configure [synchronous replication](#synchronous-replication), [Group Commit](#group-commit), [CAMO](#camo-or-commit-at-most-once), [Eager](#eager), lag control and other PGD features.
 
 #### CAMO or commit-at-most-once
 
-Wraps Eager Replication with additional transaction management at the application level to guard against a transaction being executed more than once. This transaction management is critical for high-value transactions found in payments solutions. It's roughly equivalent to the Oracle feature Transaction Guard.
+High value transactions in some applications require that the application is able to not only confirm that the transaction has been committed but that the transaction is only committed once or not at all. To ensure this happens in PGD, CAMO can be enabled allowing the application to actively participate in the transaction. 
```

Comment:
something about this first sentence keeps tripping me up 

"High value transactions in some applications require that the application is able to not only confirm that the transaction has been committed but that the transaction is only committed once or not at all. "

would this be still accurate to say? or am i misinterpreting? 

"CAMO allows your application to confirm that the transaction has been successfully committed and whether it was  only committed once or not at all. This can be especially valuable in applications that deal with high value transactions."

Reply:
Do we even need to go into the "once or not at all"? Would it be easier to grasp if we state: "successfully committed exactly once, and only once even after retries due to failover"?

("successfully committed" kinda contradicts "committed not at all")

Reply:
```suggestion
High value transactions in some applications require that the application successfully commited exactly once, and in the event of failover and retrying, only one. To ensure this happens in PGD, CAMO can be enabled allowing the application to actively participate in the transaction. 
```